querydsl-codegen/src/main/java/com/querydsl/codegen/UncapitalizedCaseTransformer.java

```diff
@@ -0,0 +1,19 @@
+package com.querydsl.codegen;
+
+import com.mysema.codegen.StringUtils;
+
+/**
+ * Uncapitalizes the first letter of a name.
+ *
+ */
+public class UncapitalizedCaseTransformer implements CaseTransformer {
+
+    /**
+     * @see com.querydsl.codegen.CaseTransformer#transform(java.lang.String)
+     */
+    @Override
+    public String transform(String name) {
```

Comment:
I think it's better to name this class 'DefaultVariableNameFunction' and make it a Guava Function<EntityType, String>.
This makes the call from EntitySerializer a bit more natural and doesn't limit the Function to just String manipulation.
What do you think?